From 32b771acbe48fcd0253ed06eba6f0149f7c48637 Mon Sep 17 00:00:00 2001
Message-Id: <32b771acbe48fcd0253ed06eba6f0149f7c48637.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Thu, 18 Sep 2014 17:16:59 +0530
Subject: [PATCH 22/75] MUST_REBASE [VPG]: adf/intel/vlv: Enable pci mmio
 remap with proper size

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed

For: GMINL-3547
Change-Id: Ic38d9575291815c6486c85810107bc0dd5b830d2
Signed-off-by: Deepak S <deepak.s@intel.com>
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>
---
 drivers/video/adf/intel/intel_adf_device.c |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/drivers/video/adf/intel/intel_adf_device.c b/drivers/video/adf/intel/intel_adf_device.c
index 8bf3369..509c335 100644
--- a/drivers/video/adf/intel/intel_adf_device.c
+++ b/drivers/video/adf/intel/intel_adf_device.c
@@ -18,7 +18,12 @@
 
 #define INTEL_ADF_DEVICE_NAME		"intel-adf-dev"
 #define INTEL_DC_REG_OFFSET		0x0
-#define INTEL_DC_REG_SIZE		0x80000
+
+#ifdef CONFIG_ADF_INTEL_VLV
+	#define INTEL_DC_REG_SIZE	(2 * 1024 * 1024)
+#else
+	#define INTEL_DC_REG_SIZE	0x80000
+#endif
 #define INTEL_VSYNC_FENCE_TIMEOUT	(5 * MSEC_PER_SEC)
 
 
-- 
1.7.9.5

